lib/membrane_h264_ffmpeg/encoder.ex

```diff
@@ -156,6 +157,19 @@ defmodule Membrane.H264.FFmpeg.Encoder do
     {:ok, %{state | encoder_ref: nil}}
   end
 
+  defp flush_encoder_if_exists(%{encoder_ref: nil}) do
+    {:ok, []}
+  end
+
+  defp flush_encoder_if_exists(%{encoder_ref: encoder_ref}) do
+    with {:ok, dts_list, frames} <- Native.flush(encoder_ref) do
+      buffers = wrap_frames(dts_list, frames)
+      {:ok, buffers}
+    else
+      {:error, reason} -> {:error, reason}
```

Comment:
This is a default with behaviour to pass the unmatched patter, so unless you really want to crash here on errors not matching the pattern `{:error, reason}` you can remove the `else` clause
```suggestion
```